### DISCUSSION TITLE
Description of the logger?

### DISCUSSION THREAD

**Discussion by aalm** [danielnilsson9/bbs-fw#136] [General]
I'd like to use a raspberry pi pico w for it instead of atmega328p. It is dualcore mcu with 2xUART in hw, and has usb+wifi too.
So should be very well suited for this, and the price difference, if any, is likely coins(pico w is < 10eur).

Is functionality of the logger described somewhere other than source?
Wondering if the "third connection" of it could be tx-only from the logger-POV?
Wondering because I have already written "udp stdio driver" for pico-sdk,
which would allow simple printf()-logging over wifi instead of usb.

**Comment by danielnilsson9** [comment]
Sounds like a good approach. Third connection was a UART -> BLUETOOTH adapter, you won't need that if you have built in BT/WIFI.
I guess you might needs level converters though since since BBS is 5V and rpi pico is 3.3V if I'm correct.

No documentation except the code (which is a bit outdated with the event log messages but should be ok otherwise).
It's just a UART proxy with interception of the eventlog messages.
